eo-maven-plugin/src/main/java/org/eolang/maven/latex/LatexTemplate.java

```diff
@@ -63,7 +61,16 @@ public String asString() {
                     new ResourceOf("org/eolang/maven/latex/latex-template.txt")
                 )
             ).asString(),
-            this.code
+            this.extractedContent()
         );
     }
+
+    private String extractedContent() {
+        return Pattern.compile("(?s)^\\s*\\+.*", Pattern.MULTILINE)
```

Comment:
@Suban05 let's move this part of code: `Pattern.compile("(?s)^\\s*\\+.*", Pattern.MULTILINE)` to private static variable so it's compiled only once.